### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by vujevits**

### CODE REVIEW

**Comment by agazso** [comment]
After cloning this repo from scratch and installing npm modules, I get the following errors:
```
npm ERR! peer dep missing: react@16.6.1, required by react-native@0.57.7
npm ERR! peer dep missing: react@^16.2.0, required by react-native-fs@2.12.1
npm ERR! peer dep missing: react@>=15.4.0, required by react-native-image-resizer@1.0.0
npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0-0 || ^16.0.0-0, required by react-redux@5.1.1
npm ERR! peer dep missing: react-native-svg@^6.5.2, required by react-native-qrcode-svg@5.1.1
npm ERR! peer dep missing: redux-persist@^4.0.0, required by redux-persist-transform-immutable@5.0.0
npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0 || ^16.0.0, required by create-react-context@0.2.2
```

File: package.json [hunk 1]
```diff
@@ -3,27 +3,22 @@
...
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "23.6.0",
     "eslint": "^4.19.1",
     "jest": "^23.6.0",
-    "jest-fetch-mock": "^1.6.5",
+    "jest-fetch-mock": "^1.6.6",
+    "metro-react-native-babel-preset": "0.48.3",
     "mocha": "^5.2.0",
-    "mock-async-storage": "^1.0.3",
-    "react-native-debugger-open": "^0.3.17",
```
Comment (agazso) [inline package.json]: Why did you remove the react-native-debugger-open? I use it to start the standalone debugger.

Comment (vujevits) [reply package.json]: I think there were a few unused modules which I removed, based on what was referenced in the code, I'll re-add this one.
